addons/edi/models/__init__.py

```diff
@@ -24,3 +24,6 @@
 from . import edi_partner_tutorial
 from . import edi_raw_document
 from . import edi_raw_record
+
+# Patching attachment behaviour
```

Comment:
Generally I would say we don't need comments in the `__init__` files unless it's something extremely unusual